test/acceptance/CustomsDeclarationCancellationUnhappyPathSpec.scala

```diff
@@ -86,7 +86,7 @@ class CustomsDeclarationCancellationUnhappyPathSpec extends AcceptanceTestSpec
       |</errorResponse>
     """.stripMargin
 
```

Comment:
typo in `BadRequestErrorXBatchIdentifierMissingOrInvalid` should be `BadRequestErrorXBadgeIdentifierMissingOrInvalid`